pallets/logion_loc/src/migration.rs

```diff
@@ -7,7 +7,7 @@ use frame_support::weights::Weight;
 use crate::{Config, File, LegalOfficerCaseOf, LocLink, LocMap, LocType, MetadataItem, pallet, PalletStorageVersion, StorageVersion};
 
 pub fn migrate<T: Config>() -> Weight {
-	do_migrate::<T, _>(StorageVersion::V2MakeLocVoid, v2::migrate::<T>)
+	do_migrate::<T, _>(StorageVersion::V2MakeLocVoid, v3::migrate::<T>)
```

Comment:
I would already introduce the new StorageVersion enum item.

Reply:
Agreed. Done [here](https://github.com/logion-network/logion-node/compare/bc3e073c953d79f38ba9fadd6174b54e4cb4a32f..f84ff7d69c6c3163ffa4e16e7ed2182a107c3084)